### DISCUSSION TITLE
More custom profile fields options

### DISCUSSION THREAD

**Discussion by Nachtkastje** [NamelessMC/Nameless#2165] [Ideas]
A function to create a full profile - Like with images, description, social media etc.
Something like a image gallery and a description about yourself would be cool

**Comment by Nachtkastje** [comment]
Maybe a new custom profile field option using the rich editor? Not sure if it would look good. @Aberdeener

**Comment by tadhgboyle** [comment]
Thanks!
I am not sure how realistic an image gallery is, but ill take a look.
I'll self assign and finish this once I am done #1764
